nservicebus/upgrades/5to6.md

```diff
@@ -262,8 +262,9 @@ This is no longer supported. We advise to, instead of using `IBuilder` directly,
 
 ### Accessing the current message context
 
-Previously it was possible to access `IMessageContext` both over `UnicastBus` and the `IBus` interface. Accessing the current message context over `UnicastBus` is no longer supported. To get access to the current message context use `IBus.CurrentMessageContext`.
+Previously it was possible to access `IMessageContext` both over `UnicastBus` and the `IBus` interface. Accessing the current message context over `UnicastBus` is no longer supported. To get access to the MessageId, the ReplyToAddress or the headers use the message handler context.
```

Comment:
@danielmarbach - Should we mention this as the `CurrentMessageContext` property as opposed to the `IMessageContext` interface? 
As users will be more familiar to the public property CurrentMessageContext?